src/pages/GetHelp.vue

```diff
@@ -265,8 +309,23 @@ export default {
         ) {
           throw new Error(this.$t('missingFields'))
         }
-
-        const [day, month, year] = this.enddate.split('.')
+        const start = this.startdate
+          .replace(' ', '/')
+          .replace(':', '/')
+          .split('/')
+        const end = this.enddate
+          .replace(' ', '/')
+          .replace(':', '/')
+          .split('/')
+        const startdate = date.formatDate(
```

Comment:
Why are you doing all this madness ?

Reply:
The problem was that I get a string from the v-model and I couldn't interpret that as a Date.
Date Handling is a bit weird in Quasar. The input string is in `DD/MM/YYYY HH:mm` but the `date`- Object is not able to interpret this.

Reply:
I used the date-fns. Thanks for the hint!